.gitignore

```diff
@@ -8,5 +8,6 @@
 **/*/go.sum
 **/.project
```

Comment:
thanks for deleting .project. Can you also remove this line here `**/.project` but leave the other one